app/src/main/java/com/nextcloud/talk/controllers/CallController.java

```diff
@@ -1958,6 +1970,7 @@ private void setupAvatarForSession(String session) {
                         avatarImageView.setController(draweeController);
                     }
                 }
+                setCallState(CallStatus.IN_CONVERSATION);
```

Comment:
- [ ] discuss with @tobiasKaminsky if this is allowed here and if deleting in line 1588 was okay

Reply:
As far as I understand it, you postpone setting call state "in_conversation" a bit later, so that you are sure that already a proper connection to at least one recipient is available.
I think it is fine :+1:

Reply:
at least deleting in line 1588 was not okay and introdcued new bugs. anyway, busy with other things right now....